### PR TITLE
 Implement "New snippet" page styles

### PR DESCRIPTION
**Pull request by lotrien**

### CODE REVIEW

File: src/components/NewSnippet.jsx [hunk 1]
```diff
@@ -1,10 +1,60 @@
...
+    <Title title="New snippet" key="title" />,
+    <div className="new-snippet" key="new-snippet">
+      <div className="new-snippet-code-wrapper">
+        <div className="new-snippet-code-header">
+          <Input placeholder="Title" />
+          <Input placeholder="Tags (separate tags by comma)" />
+        </div>
+        <div className="new-snippet-code">
+          <CodeMirror
+            value="console.log('All heil XSnippet')"
```
Comment (malor) [inline src/components/NewSnippet.jsx]: nit: `hail`, otherwise it's in German ;)

File: src/components/NewSnippet.jsx [hunk 2]
```diff
@@ -1,10 +1,60 @@
...
+            options={{ lineNumbers: false }}
```
Comment (malor) [inline src/components/NewSnippet.jsx]: Why not show the line numbers?

File: src/components/NewSnippet.jsx [hunk 3]
```diff
@@ -1,10 +1,60 @@
...
+  [
+    <Title title="New snippet" key="title" />,
+    <div className="new-snippet" key="new-snippet">
+      <div className="new-snippet-code-wrapper">
+        <div className="new-snippet-code-header">
+          <Input placeholder="Title" />
+          <Input placeholder="Tags (separate tags by comma)" />
+        </div>
+        <div className="new-snippet-code">
+          <CodeMirror
```
Comment (malor) [inline src/components/NewSnippet.jsx:22]: Hmm, I gave it a try and looks like the code input is not editable. Is that intentional?

File: src/components/NewSnippet.jsx [hunk 4]
```diff
@@ -1,10 +1,60 @@
...
+        </div>
+      </div>
+      <div className="new-snippet-lang-wrapper">
+        <div className="new-snippet-lang-header">
+          <Input placeholder="Type to search..." />
+        </div>
+        <div className="new-snippet-lang-list-wrapper">
+          <Scrollbars>
+            <ul className="new-snippet-lang-list">
+              <li className="new-snippet-lang-item">Text1</li>
```
Comment (malor) [inline src/components/NewSnippet.jsx:38]: I'll add a new route to the API to retrieve this.

**Review by malor — COMMENTED** [review]
A few questions inline. Overall looks good to me. 👍

File: src/components/NewSnippet.jsx (hunk 3 above)
Comment (lotrien) [reply src/components/NewSnippet.jsx:22]: Yeap, for now it is not editable as this is just stubbed data inside

File: src/components/NewSnippet.jsx (hunk 1 above)
Comment (lotrien) [reply src/components/NewSnippet.jsx]: ;) will fix

File: src/components/NewSnippet.jsx (hunk 2 above)
Comment (lotrien) [reply src/components/NewSnippet.jsx]: According to old UI and new one in design there are no line numbers on New Snippet page, (you could see line numbers on Snippet page)

File: src/components/NewSnippet.jsx (hunk 4 above)
Comment (lotrien) [reply src/components/NewSnippet.jsx:38]: Great, but it's not urgent. I just thought that I will create all markup with stabbed data at first  and only then divide into small component and add logic

**Review by ikalnytskyi — APPROVED** [review]
Под пиво покатит.

**Pull request merged**